src/main/java/com/nike/cerberus/command/certificates/DeleteOldestCertificatesCommand.java

```diff
@@ -33,6 +34,14 @@
 
     public static final String COMMAND_NAME = "delete-oldest-certificates";
```

Comment:
Is there protection against deleting all Certificates?

Reply:
Yes, is Runnable checks that there are at least 2 certs, and the delete logic ignores the newest cert.